content/index.md

```diff
@@ -42,6 +42,7 @@ Run it locally:
 
 ```bash
 cd myapp
+npm install
```

Comment:
I think this should be `meteor npm install` if it's in the getting started section, since some people might not have NPM installed globally.